pkg/virt-controller/watch/topology/tsc.go

```diff
@@ -102,8 +113,10 @@ func CalculateTSCLabelDiff(frequenciesInUse []int64, frequenciesOnNode []int64,
 		}
 	}
 
-	for _, freq := range frequenciesInUse {
-		if freq <= nodeFrequency {
+	for freq := range requiredMap {
+		// For the non-scalable case, the map was already sanitized above.
+		// For the scalable case, we can accept all values that are either lower than us or within the tolerance range
```

Comment:
Not on this comment:

> For the scalable case, we can accept all values that are either lower than us or within the tolerance range

Instead:

> For the scalable case, we can accept all node frequencies that are either higher than the current cluster frequency or within the tolerance range

Reply:
Looks like we accept all node frequencies that are _lower_ than the current cluster frequency, not higher, right?